web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed.md

```diff
@@ -0,0 +1,22 @@
+#Guest Agent not installed
+
+The purpose of this document is to show what a VM would look like without the guest agent installed. 
+
+In the list view we show an “info” icon with the status with a tooltip that show that the guest agent isn’t installed. Further information can be provided in a tooltip.
+
+For the IP address, when the GA ins't installed, items will get “unavailable” text and a tooltip with more info on hover.
```

Comment:
Small typo on "isn't".

Reply:
Might also want to un-abbreviate "GA" to "guest agent" - I got a little confused for a second.

Reply:
Might also want to un-abbreviate "GA" to "guest agent" - I got a little confused for a second.